### ISSUE TITLE
Improve aria-label values in non-text links in the StoryActions component

### ISSUE THREAD

**Issue by julianguyen** [ifmeorg/ifme#2121]
# Description

The [Story component](http://design.if-me.org/?path=/docs/components-story--all-props) ([code](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Story/index.jsx)) appears on many pages, including the signed-in homepage when there are viewable Moments or Strategies and any signed-in page that's displaying a list of items.

The [StoryActions sub-component](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Story/StoryActions.jsx) has non-text links (icons with tooltips) for functionality like editing and deleting. There are existing values for `aria-label` on these non-text links, but they should be improved.

Let's include the name of the story that the action is being applied to. The [change should be made here](https://github.com/ifmeorg/ifme/blob/main/client/app/components/Story/StoryActions.jsx#L120), and we should concatenate the prop `name` from `Story` to this string. 

e.g. "Edit Story-Name-Here"

We'll have to account for internationalization (i18n) for different grammar structures in different languages. Feel free to defer to Google Translate for verification.

---

Please assign yourself (via the Assignees dropdown), if you do want to work on this issue. Can't find yourself? You need to [join our organization](https://github.com/ifmeorg/ifme/wiki/Join-Our-Slack).

Check out our [Picking Up Issues](https://github.com/ifmeorg/ifme/wiki/Picking-Up-Issues) guide if you haven't already!
